Makefile

```
@@ -1,19 +1,8 @@
-# Check if GPU is available
-NVCC_RESULT := $(shell which nvcc 2> NULL)
-NVCC_TEST := $(notdir $(NVCC_RESULT))
-ifeq ($(NVCC_TEST),nvcc)
-GPUS=--gpus all
-else
-GPUS=
-endif
-
-# For Windows use CURDIR
-ifeq ($(PWD),)
-PWD := $(CURDIR)
-endif
+# Check if GPU is available - if `nvidia-smi` works then use GPUs
+GPUS := $(shell command -v nvidia-smi > /dev/null && nvidia-smi > /dev/null 2>&1 && echo "--gpus all" || echo "")
 
 # Set flag for docker run command
-BASE_FLAGS=-it --rm  -v ${PWD}:/home/app/mava -w /home/app/mava
+BASE_FLAGS=-it --rm -v ${PWD}:/home/app/mava -w /home/app/mava
```

I think we don't have to use volumes. 

```suggestion
BASE_FLAGS=-it --rm
```